core/services/workflow_service/schemas/compute_block.py

```diff
@@ -9,8 +9,11 @@
     InputOutput,
     InputOutputType
 )
+from services.workflow_service.schemas.workflow import BlockStatus
 from utils.config.environment import ENV
 
+ConfigType = Dict[str, Optional[Union[str, int, float, List, bool]]]
```

Comment:
This is btw old typing style. Use `| None` instead of `Optional` and `list` instead of `List` same for `dict` and so on.